### PR TITLE
Updates repl module to accept a full discord message's worth of chara…

### PR DESCRIPTION
**Pull request by Cisien**
…cters.

### CODE REVIEW

**Comment by Cisien** [comment]
Ready to go

File: Modix.Bot/Modules/ReplModule.cs [hunk 1]
```diff
@@ -91,16 +82,8 @@ public class ReplModule : ModuleBase
...
 
-                using (var tokenSrc = new CancellationTokenSource(30000))
-                {
-                    res = await client.PostAsync(_replUrl, content, tokenSrc.Token);
-                }
-            }
-            catch (TaskCanceledException)
-            {
-                await ModifyOrSendErrorEmbed("Gave up waiting for a response from the REPL service.", message);
-                return;
```
Comment (Scott-Caldwell) [inline Modix.Bot/Modules/ReplModule.cs]: Is this safe to remove?

Comment (Cisien) [reply Modix.Bot/Modules/ReplModule.cs]: Yep, repl gives up after 30 seconds anyway, this check was never actually hit in practice.

Comment (Scott-Caldwell) [reply Modix.Bot/Modules/ReplModule.cs]: But does MODiX give up after 30 seconds?

Comment (Cisien) [reply Modix.Bot/Modules/ReplModule.cs]: it fails with a 502 because the repl process was killed after 30 seconds.

Comment (Cisien) [reply Modix.Bot/Modules/ReplModule.cs]: image

**Pull request merged**